sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/AzureMonitorOptions.cs

```diff
@@ -59,6 +59,16 @@ public class AzureMonitorOptions : ClientOptions
         /// </summary>
         public string StorageDirectory { get; set; }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AzureMonitorOptions"/>.
+        /// </summary>
+        public AzureMonitorOptions()
+        {
+            // users can explicitly change it, but by default we don't want self-diagnostics to be reported to Azure Monitor.
+            this.Diagnostics.IsDistributedTracingEnabled = false;
+            this.Diagnostics.IsLoggingEnabled = false;
```

Comment:
with this, is there a need to do the suppress.enter for livemetrics/exporter etc.? Or we need it to prevent the underlying http calls itself from being captured as Spans by the instr. library?

Reply:
great point. Yes,  I believe we still need to suppress generic HTTP instrumentation as well - `this.Diagnostics.IsDistributedTracingEnable` cannot do it. I also don't know what are all the things that need to be suppressed.